data/schema/types.yaml

```diff
@@ -8,7 +8,7 @@ Euro:
   data_type: int|float
   unit: EUR
   type: Unit
-Gravimetry:
+Gravimetric:
```

Comment:
Isn't this an adjective? Why changing from `Gravimetry`?

Any docs on this?

Reply:
It's short hand for 'Gravimetric Units' as that is the actual name of the unit (equivalent to `Euros`). `Gravimetry` is a technique for measuring something (equivalent to saying something like `Accounting`)

If we want to be completely precise we should just call it `GravimetricUnits`

Reply:
I'd use the term used in standards rather than a short hand